src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -75,6 +70,16 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
     }
 
     /**
+     * Return the names of the classes mapped in this file.
+     *
+     * @return string[] The classes names
+     */
+    public function getMappedClasses()
```

Comment:
I think the best option is to wait for someone to express the need. Creating an interface is a quite important choice as we will have to maintain it for BC.